components/eam/src/physics/cam/cam_diagnostics.F90

```diff
@@ -387,6 +387,24 @@ subroutine diag_init()
    call addfld ('TH9251000',horiz_only,   'A','K','Theta difference 925 mb - 1000 mb')   
    call addfld ('THE9251000',horiz_only,   'A','K','ThetaE difference 925 mb - 1000 mb') 
 
+   ! Add in fields for T and U (not already included) to track Sudden Stratospheric Warming events
+   ! Levels include: 250, 200, 150, 100, 50, 25, 10, 5, 2, 1, and TOP (numbers in hPa)
+   call addfld ('T250',horiz_only,   'A', 'K',   'Temperature at 250 mbar pressure surface')
+   call addfld ('T150',horiz_only,   'A', 'K',   'Temperature at 150 mbar pressure surface')
+   call addfld ('T050',horiz_only,   'A', 'K',   'Temperature at 50 mbar pressure surface')
+   call addfld ('T025',horiz_only,   'A', 'K',   'Temperature at 25 mbar pressure surface')
+   call addfld ('T005',horiz_only,   'A', 'K',   'Temperature at 5 mbar pressure surface')
+   call addfld ('T002',horiz_only,   'A', 'K',   'Temperature at 2 mbar pressure surface')
+   call addfld ('T001',horiz_only,   'A', 'K',   'Temperature at 1 mbar pressure surface')
+   call addfld ('TTOP',horiz_only,   'A', 'K',   'Highest model level temperature')
+   call addfld ('U150',horiz_only,   'A', 'm/s', 'Zonal wind at 150 mbar pressure surface')
+   call addfld ('U050',horiz_only,   'A', 'm/s', 'Zonal wind at 50 mbar pressure surface')
```

Comment:
@beharrop cam we add U250 as well to correspond to T250?

Reply:
@vanroekel , we have it.  T200, U250, U200, U010 already exist so not showing up in the changes. The test @beharrop did above (in fincl2) also confirmed that.